### DISCUSSION TITLE
Github logo instead rank level

### DISCUSSION THREAD

**Discussion by AryanKarumuri** [anuraghazra/github-readme-stats#3519] [Q&A] [answered]
How to get Github logo instead rank level? Could anyone help? Thanks in advance.

**Comment by qwerty541** [comment, marked as answer]
Hey, @AryanKarumuri! You can show GitHub logo instead of rank level on your stats card using `&rank_icon=github` querystring parameter.

### Before

```html
<img src="https://github-readme-stats.vercel.app/api?username=aryankarumuri&show_icons=true&locale=en&theme=midnight-purple" />
```

<img src="https://github-readme-stats.vercel.app/api?username=aryankarumuri&show_icons=true&locale=en&theme=midnight-purple" />

### After

```html
<img src="https://github-readme-stats.vercel.app/api?username=aryankarumuri&show_icons=true&locale=en&theme=midnight-purple&rank_icon=github" />
```

<img src="https://github-readme-stats.vercel.app/api?username=aryankarumuri&show_icons=true&locale=en&theme=midnight-purple&rank_icon=github" />

**Comment by AryanKarumuri** [comment]
Hi @qwerty541 
I'm grateful for your assistance. It worked perfectly.

**Comment by Jahnavi-57** [comment]
Thanks a lot. It worked!

**Reply by qwerty541** [reply]
Glad to help! Feel free to contact me if you need any further assistance.
